pkg/cmd/serviceaccount/list/list.go

```diff
@@ -90,36 +75,6 @@ func runList(opts *options) (err error) {
 		return nil
 	}
 
-	outStream := opts.IO.Out
-	switch opts.output {
-	case dump.JSONFormat:
-		data, _ := json.MarshalIndent(res, "", cmdutil.DefaultJSONIndent)
-		_ = dump.JSON(outStream, data)
-	case dump.YAMLFormat, dump.YMLFormat:
-		data, _ := yaml.Marshal(res)
-		_ = dump.YAML(outStream, data)
-	default:
-		rows := mapResponseItemsToRows(serviceaccounts)
-		dump.Table(outStream, rows)
-	}
-
+	dump.PrintDataInFormat(opts.output, res, opts.IO.Out)
 	return nil
 }
-
-func mapResponseItemsToRows(svcAccts []kafkamgmtclient.ServiceAccountListItem) []svcAcctRow {
-	rows := []svcAcctRow{}
-
-	for _, sa := range svcAccts {
-		row := svcAcctRow{
-			ID:        sa.GetId(),
-			Name:      sa.GetName(),
-			ClientID:  sa.GetClientId(),
-			Owner:     sa.GetOwner(),
-			CreatedAt: sa.GetCreatedAt().String(),
-		}
-
-		rows = append(rows, row)
-	}
-
-	return rows
-}
```

Comment:
What is the reason for deleting this?

Reply:
I thought the only acceptable formats were specified in ValidOutputFormats? I see nothing there about table format.

`ValidOutputFormats       = []string{dump.JSONFormat, dump.YAMLFormat, dump.YMLFormat}`

Reply:
Table format is used when there is no format passed in the list commands (default). There is no way so specify it using the output format.

Reply:
@craicoverflow done